assets/src/application/services/toaster/Toaster.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { ToasterProps } from './types';
+
+const Toaster: React.FC<ToasterProps> = ({ icon, message = 'loading...' }) => {
+	return (
+		<div className='ee-toaster-notice__toast-body'>
```

Comment:
I've borrowed some class names from here: https://fkhadra.github.io/react-toastify/how-to-style